docs/src/extend/shareable-configs.md

```diff
@@ -66,59 +72,62 @@ Be sure to replace `eslint-config-myconfig` with the actual name of your module.
 
 ## Using a Shareable Config
 
-Shareable configs are designed to work with the `extends` feature of `.eslintrc` files. Instead of using a file path for the value of `extends`, use your module name. For example:
+To use a shareable config, include it in the `extends` field of an `.eslintrc` file. For the value, use your module name. For example:
 
 ```json
 {
     "extends": "eslint-config-myconfig"
 }
 ```
 
-You can also omit the `eslint-config-` and it will be automatically assumed by ESLint:
+You can also omit the `eslint-config-` and it is automatically assumed by ESLint:
 
 ```json
 {
     "extends": "myconfig"
 }
 ```
 
-### npm scoped modules
+### npm Scoped Modules
 
 npm [scoped modules](https://docs.npmjs.com/misc/scope) are also supported in a number of ways.
 
-By using the module name:
+You can use the module name:
 
 ```json
 {
     "extends": "@scope/eslint-config"
 }
 ```
 
-You can also omit the `eslint-config` and it will be automatically assumed by ESLint:
+You can also omit the `eslint-config` and it is automatically assumed by ESLint:
 
 ```json
 {
     "extends": "@scope"
 }
 ```
 
-The module name can also be customized, just note that when using [scoped modules](https://docs.npmjs.com/misc/scope) it is not possible to omit the `eslint-config-` prefix. Doing so would result in package naming conflicts, and thus in resolution errors in most of cases. For example a package named `@scope/eslint-config-myconfig` vs `@scope/myconfig`, since both are valid scoped package names, the configuration should be specified as:
+The module name can also be customized, just note that when using [scoped modules](https://docs.npmjs.com/misc/scope) you cannot omit the `eslint-config-` prefix. Doing so results in package naming conflicts, and thus often in resolution errors. For example, if you have a package named `@scope/eslint-config-myconfig`, the configuration must be specified as:
 
 ```json
 {
     "extends": "@scope/eslint-config-myconfig"
 }
```

Comment:
This isn't true, a `@scope/eslint-config-myconfig` package can be used by specifying `"extends": "@scope/myconfig"`. The only case where `"eslint-config"` sequence must appear in the config specifier is when specifying a subpath of a `@scope/eslint-config` package, like in the example on line 150 (`"extends": "@scope/eslint-config/my-special-config"`), because `"@scope/my-special-config"` would be interpreted as `"@scope/eslint-config-my-special-config"`.

Reply:
You're correct. I think this is a case where the old docs were also incorrect, so probably a good idea to clean this up.

Reply:
ok, clarifying the text and examples